cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,84 @@
+import { generateUser } from '../support/generate';
 /// <reference types='cypress' />
 
+const user = generateUser();
+
+const rowsChoose = user.rows[Math.floor(Math.random() * user.rows.length)];
+
+const worker = 'Cierra';
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('pagination testing', () => {
+    cy.get('.-pageInfo').should('contain', 'Page');
+
+    cy.get('select').select(rowsChoose);
+
+    for (let i = 0; i < parseInt(rowsChoose.split(' ')[0]); i++) {
+      cy.get(`:nth-child(${i + 1})`).should('exist');
+    }
+  });
```

Comment:
what do you test here?

Reply:
If the user chooses 25 rows it will be tested, so that all 25 rows are exist

Reply:
check pls this loop on ui mode) I cant understand what you test here. You need to click buttons and assert that pages changes. Also need to check row count selection

Reply:
It checks the number of rows, if the user chooses 5 rows, it will be checked, that all rows exist. 😥
And I fixed the side menu checking.
Also, I added the assertion that page changes.